app/controllers/users_controller.rb

```diff
@@ -0,0 +1,2 @@
+class UsersController < ActionController::Base
```

Comment:
Why do we need to add an empty controller? Devise's own controller should take care of login routes and registration routes. 🤔

Reply:
Ok, so I'm going to delete it